common/types.go

```diff
@@ -49,7 +49,6 @@ const (
 	NumRegionsInPrime = 3
 	NumZonesInRegion  = 3
 	HierarchyDepth    = 3
-	NumChains         = 1 + NumRegionsInPrime*(1+NumZonesInRegion) // Prime + R regions + RxZ zones
```

Comment:
Why was this and subsequent references to NumChains removed?

Reply:
this commit just reverts the wizeguyys change to the bucket size, we need to rethink about these numbers and decide what to do

Reply:
Ok pinging @wizeguyy to weigh in if possible